front/src/views/modals/CreateTableModal.vue

```diff
@@ -113,6 +112,8 @@ export default Vue.extend({
     async confirmCreateTableModal () {
       try {
         const createTableResponse = await lckServices.table.create({
+          // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+          // @ts-ignore
```

Comment:
why did you use `@ts-ignore` ?

Reply:
Well, I got some errors with eslint + typescript.
So, the final solution was to add a `@ts-ignore`. Ugly, I agree, but I don't have another solution...